readme.md

```diff
@@ -164,20 +164,47 @@ Note:
 
 For more details please refer [these](https://github.com/Azure/autorest/tree/master/docs/generate) docs to generate code from your OpenAPI definition using AutoRest.
 
-## Debugging 
+## Debugging
 
-There are two entry points for debugging autorest.csharp:
+### Debugging projects inside the `autorest.csharp` repo
 
-- Opening AutoRest.CSharp.sln in Visual Studio will contain a debugging profile for each sample and test. These only run the "back half", using the yaml file generated by autorest.core as input.
-- ./Generate.ps1 will execute the entire pipeline by invoking autorest and including autorest.csharp as a plugin. Add:
+There are two entry points for debugging the generator:
 
-```
+- Opening `AutoRest.CSharp.sln` in Visual Studio will contain a debugging profile for each sample and test. These only run the "back half", using the yaml file generated by `autorest.modelerfour` as input.
```

Comment:
```suggestion
- Opening `AutoRest.CSharp.sln` in Visual Studio will contain a debugging profile called 'launchsetting.json' for each sample and test. These only run the "back half", using the yaml file generated by `autorest.modelerfour` as input.
```

Reply:
I do not want to mention this file. This file is "how the debugging profile is stored on the disk". Here we are trying to state that there is a debugging profile, and it works with Visual Studio only.
If the reader is familiar with how Visual Studio works, they should know there is a file `launchsettings.json`.

On the other hand, these statements are confusing. I am planning to rewrite them